cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,55 @@
 describe('', () => {
   before(() => {
+    describe('Article page', () => {
+      let user;
+      let article;
 
-  });
-
-  it('', () => {
+      beforeEach(() => {
+        cy.task('generateUser').then((generateUser) => {
+          user = generateUser;
+        });
+        cy.task('generateArticle').then((generateArticle) => {
+          article = generateArticle;
+        });
+      });
 
-  });
+      it('', () => {
+        it('should allow to create an article', () => {
+          cy.login(user.email, user.username, user.password);
```

Comment:
You put a block 'it' within a block 'it' - it should be only one. there is no need for this duplication. Check the same in the article deletion test